examples/platform/silabs/efr32/efr32_utils.h

```diff
@@ -33,6 +33,14 @@ void appError(int err);
 #ifdef __cplusplus
 }
 
+// Output logs to RTT by defaults
+#ifndef SILABS_LOG_OUT_UART
+#define SILABS_LOG_OUT_UART 0
+#endif
+#ifndef SILABS_LOG_OUT_RTT
```

Comment:
I believe in the default case (when --uart_log isn't used) this isn't defined project-wide and it is not seen by `logging.cpp`. Thus RTT wouldn't work and would explain the ci failure

Reply:
It works with the default use case, but for the Unit test it doesn't since the AppConfig.h use isn't the same as for the other examples...